src/core/loss_detection.c

```diff
@@ -456,6 +463,32 @@ QuicLossDetectionOnPacketSent(
             &Connection->CongestionControl, SentPacket->PacketLength);
     }
 
+    SentPacket->Flags.IsAppLimited = QuicCongestionControlIsAppLimited(&Connection->CongestionControl);
```

Comment:
I see you set the IsAppLimited flag here before potentially calling QuicCongestionControlSetAppLimited. Should this line instead come afterward?

It's very hard for me to answer this question for myself because there's no comment in QUIC_SEND_PACKET_FLAGS describing what the IsAppLimited flag means exactly. In general this PR needs to add comments to the new struct fields describing what they are for.

Reply:
Sure, will do.

Reply:
You neither added a comment to QUIC_SEND_PACKET_FLAGS nor did you move this IsAppLimited flag assignment after the QuicCongestionControlSetAppLimited call. Why was this thread marked resolved?

Reply:
my bad, my bad... I thought you were mentioning the comments in `LAST_ACKED_PACKET_INFO`.

I think now both have been fixed.